### PR TITLE
implemented two_sum and added tests

### PR DESCRIPTION
**Pull request by sophiat8832**
PLEASE LOOK AT MY CODE T-T

### CODE REVIEW

File: tests/two_sum_test.py [hunk 1]
```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.two_sum import twoSum
+
+def test_returns_0_1_for_2_7_11_15_with_target_9():
```
Comment (aruaaa17) [inline tests/two_sum_test.py:4]: Plz rename the text🤣

**Review by aruaaa17 — CHANGES_REQUESTED** [review]
Hiiiiii

File: tests/two_sum_test.py (hunk 1 above)
Comment (sophiat8832) [reply tests/two_sum_test.py:4]: NO

**Review by aruaaa17 — CHANGES_REQUESTED** [review]
Hey!!!

**Review by jping0220 — COMMENTED** [review]
everything looks good

**Review by jping0220 — COMMENTED** [review]
it's good

**Pull request merged**